flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -474,6 +566,20 @@ def bash(command):
 def create_resume_out_filename(
     flepi_run_index: str, flepi_prefix: str, flepi_slot_index: str, flepi_block_index: str, filetype: str, liketype: str
 ) -> str:
+    """
+    Compiles run output information.
+
+    Args:
+        flepi_run_index: Index of the run (str).
+        flepi_prefix: File prefix (str).
+        flepi_slot_index: Index of the slot (str).
+        flepi_block_index: Index of the block (str).
+        filetype: File type (str).
+        liketype: Chimeric or global (str).
```

Comment:
The "(str)" type hints are no longer needed since they're in the function definition.